include/internal/catch_default_main.hpp

```diff
@@ -13,12 +13,20 @@
 
 #ifndef __OBJC__
 
+#ifndef CDECL
```

Comment:
I don't like unnamespaced macros, so I would prefer this to be

```suggestion
#ifndef CATCH_INTERNAL_CDECL
```

Reply:
Sure; done.